app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/BaseActivity.java

```diff
@@ -99,4 +107,28 @@ final void initBackToolbar(Toolbar toolbar) {
             bar.setHomeButtonEnabled(true);
         }
     }
+
+    @SuppressWarnings("deprecation")
+    private Context setUpLocale(Context context, String language) {
```

Comment:
I believe this method should be kept in LocaleUtil.java because this is for "Locale" update.